### ISSUE TITLE
Error in album controller in zip detection

### ISSUE THREAD

**Issue by FUSAKLA** [Ruplahlava/Rupsha#15]
line 80 in file /application/controllers/Album.php 
passes wrong arguments to zip_exists function

actual state:

```
$this->data['zip_download'] = $this->download_zip->zip_exists($album[0]->id_user,$aUser[0]->login);
```

should be changed to:

```
$this->data['zip_download'] = $this->download_zip->zip_exists($album[0]->id);
```

**Comment by Ruplahlava** [comment]
Hello, already fixed it, forgot to push it...

**Comment by Ruplahlava** [comment]
Fixed by tag 0.1.10a
https://github.com/Ruplahlava/Rupsha/commit/68ba20e65753ac8a8a5ec5476355465338671a6f
